persistence/src/test/java/com/epam/rft/atsy/persistence/repositories/UserRepositoryIT.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.rft.atsy.persistence.repositories;
+
+import com.epam.rft.atsy.persistence.entities.UserEntity;
+import org.hamcrest.CoreMatchers;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.nullValue;
+
+public class UserRepositoryIT extends AbstractRepositoryIT {
+
+    private final String UNREGISTERED_USER_NAME = "unregistered";
+    private final String UNREGISTERED_USER_PASSWORD = "not_a_pass";
+    private final String REGISTERED_USER_NAME = "Test";
+    private final String REGISTERED_USER_PASSWORD = "$2a$04$6r2eBlucnE3/m0fDYgE3e.eI5akUx55qPMID3O/SlptCIqOFFcCPK";
+
+    @Autowired
+    UserRepository repository;
```

Comment:
Please add the private keyword, because I bet you don't want others to instantiate this class and access the repository in it.